src/rail/cli/rail_pipe/pipe_scripts.py

```diff
@@ -16,6 +16,24 @@
 from rail.cli.rail_pipe.pipe_options import RunMode
 
 
+S3DF_SLURM_OPTIONS: list[str] = [
+    "-p",
+    "milano",
+    "--account",
+    "rubin:commissioning@milano",
+    "--mem",
+    "16448",
```

Comment:
you can write this as 16G, btw (don't need to change, but may be a little cleaner to read)